docs/usecases/datastorage_for_institutions.rst

```diff
@@ -0,0 +1,314 @@
+the advantage of vastly simplified version control, even for trainees, and
+simplified access to projects for collaborators and supervisors. Data
+from the data store is installed as subdatasets. This comes with several
+benefits: Analyses are automatically linked to data, no unused file
+copies waste disk space on the compute cluster as data can be retrieved
+on-demand, and files that are easily re-obtained or recomputed can safely be
+dropped to save even more disk-space. More over, upon creation of an analysis
+project, the associated Gitlab project it is automatically configured as a remote
+with a publication dependency on the data store, thus enabling vastly simplified
+data publication routines and back-up of pristine results.
+
+
+Step-by-step
+^^^^^^^^^^^^
+
+.. note::
+
+   This use case describes the data storage implementation as done in INM-7,
+   research centre Juelich, Germany.
+
+To create a data store, parts of the old compute cluster and parts of the
+super computer at the Juelich supercomputing centre (JSC) are used to store
+large amounts of data. Thus, multiple different, independent machines take care of
+warehousing the data. While this is unconventional, it is convenient: The
+data does not strain the compute cluster, and with DataLad, it is irrelevant
+where the data is located.
+
+.. figure:: ../artwork/src/ephemeral_infra.svg
+   :alt: A simple, local version control workflow with datalad.
+   :figwidth: 80%
+
+   Trinity of research data handling: The data store (``$DATA``) is managed and
+   backed-up. The compute cluster (``$COMPUTE``) has an analysis-appropriate structure
+   with adequate resources, but just as users workstations/laptops (``$HOME``),
+   it is not concerned with data hosting.
+
+On their own machines (``$HOME``), researchers are free to do whatever they want
+as long as it is within the limits of their machines. The cluster (``$COMPUTE``)
+pulls the data exclusively from the data store (``$DATA``). Thus, within
+``$HOME``, researchers are free to explore data from ``$DATA`` as they wish,
+but scaling requires them to use ``$COMPUTE``. Results from ``$COMPUTE`` are pushed
+back to ``$DATA``, and hence anything that is relevant for a computation is tracked
+(and backed-up) there.
+
+The data store as a Git-annex RIA remote
+""""""""""""""""""""""""""""""""""""""""
+
+The distributed data store exists thanks to Git-annex: Any large file content in
+datasets is stored as a *value* in Git-annex's object tree. A *key*
+generated from its contents is checked into Git and used to reference the
+location of the value in the object tree [#f1]_. The object tree (or *keystore*)
+with the data contents can be located anywhere -- its location only needs to be
+encoded using a *special remote*. This configuration is done on an
+administrative, system-wide level, and users do not need to care or know
+about where data is stored, as they can access it just as easily as before.
```

Comment:
Eh. In all honesty: it's not as easy as before. The data was always just there. Now they have to issue commands and wait for it to transfer.

It is all a step backwards for users until they finally can do something that they couldn't before. And at this point, they aren't. Those gains come with the new system.

Perhaps it's as simple as performing their jobs without incurring the wrath of their local sysadmin.... but still. ;-)